HealthcareSystem/GraphicalEditor/Enumerations/MapObjectDepartments.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace GraphicalEditor.Enumerations
 {
-    public enum MapObjectDepartment
+    public enum MapObjectDepartments
```

Comment:
Check for [this](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations) and correct it.

Reply:
I understand your comment and judging by it, the name of the enumeration should be singular type name, but I don't know how I would then name the class whose field is of the type of that enumeration.

Reply:
I have renamed Enumerations according to C# conventions, so check them out now. 😄

Reply:
Much better, well done!